cmd/network-controller/main.go

```diff
@@ -98,17 +104,11 @@ func run(c *cli.Context, registerFuncList []config.RegisterFunc, leaderelection
 	helperImage := c.String("helper-image")
 
 	if threadiness <= 0 {
-		klog.Infof("Can not start with thread count of %d, please pass a proper thread count.", threadiness)
-		return nil
+		klog.Infof("Thread count of %d is invalid, fallback to default value 2.", threadiness)
+		threadiness = 2
```

Comment:
Nit: consider making it a const since it's also used in the CLI flag definition.

Reply:
Good idea, updated.